### PR TITLE
Create tag component

### LINKED ISSUE #5: Tag Component
As a user
i want to know which gamemode is associated with a specific content
for that i need tags

- [x] Prop for the text
- [x] Prop for the color
- [x] Prop for onClick

### PR DESCRIPTION
**Pull request by ChristianSeelemann**
Deployment: https://fortknights-component-t-e0aawq.herokuapp.com/storybook/?path=/story/component-tag--battle-royale

Just a very tiny tag component. Nothing special. Mostly copied from the button component.

- Prop for the text
- Prop for the color
- Prop for onClick

### CODE REVIEW

File: src/app/components/Tag/Tag.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import styles from './Tag.module.css';
+
+interface TagProps {
+  style: 'br' | 'stw' | 'creative' | 'date';
```
Comment (marwinburesch) [inline src/app/components/Tag/Tag.tsx]: breakout room? star wars?
I think coming up with a more descriptive term would do a lot here :)

File: src/app/components/Tag/Tag.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Tag from './Tag';
+
+export default {
+  title: 'Component/Tag',
+  component: Tag,
+};
+
+export const BattleRoyale = (): JSX.Element => (
+  <Tag style="br" text="Battle Royale" />
```
Comment (marwinburesch) [inline src/app/components/Tag/Tag.stories.tsx]: AAAAAAAH, hm but still

**Review by marwinburesch — COMMENTED** [review]
Awesome

File: src/app/components/Tag/Tag.stories.tsx (hunk 2 above)
Comment (ChristianSeelemann) [reply src/app/components/Tag/Tag.stories.tsx]: Solved =)

**Pull request merged**